aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/services/ApplicationService.java

```diff
@@ -24,6 +27,12 @@
 public interface ApplicationService {
     ApiInfoDto getAipConsoleApiInfo();
 
+    int fastScan(FastScanProperties fastScanProperties) throws JobServiceException, PackagePathInvalidException, UploadException;
+
+    int deepAnalyze(DeepAnalyzeProperties properties) throws JobServiceException;
+
+    int publishToImaging(String applicationName, long sleepDuration, boolean verbose, LogPollingProvider logPollingProvider);
+
```

Comment:
Application Service is not running for Jobs, its for getting information about Applications.

If you want to run a Job, you create new methods in JobsService.

Reply:
This is dealing with an application even though is triggering one or multiple jobs.
If jobService is required to do some stuff jobService will be updated accordingly.
But still, the entry point to deal with an application is applicationService